src/cascadia/TerminalApp/defaults.json

```diff
@@ -209,6 +209,8 @@
         { "command": "switchToTab5", "keys": ["ctrl+alt+6"] },
         { "command": "switchToTab6", "keys": ["ctrl+alt+7"] },
         { "command": "switchToTab7", "keys": ["ctrl+alt+8"] },
-        { "command": "switchToTab8", "keys": ["ctrl+alt+9"] }
+        { "command": "switchToTab8", "keys": ["ctrl+alt+9"] },
+		{ "command": "decreaseFontSize", "keys": ["ctrl+-"] },

```

Comment:
```suggestion
        { "command": "decreaseFontSize", "keys": ["ctrl+-"] },
```

Reply:
Good catch. Shouldn't have to tell me this 😅. Fixed.